src/Juvix/Compiler/Backend/VampIR/Pretty/Options.hs

```diff
@@ -2,14 +2,17 @@ module Juvix.Compiler.Backend.VampIR.Pretty.Options where
 
 import Juvix.Prelude
 
--- no fields for now, but make it easier to add options in the future I don't
--- remove this datatype entirely
-data Options = Options
+newtype Options = Options
+  { _optIntegerBits :: Int
+  }
 
 makeLenses ''Options
 
 defaultOptions :: Options
-defaultOptions = Options
+defaultOptions =
+  Options
+    { _optIntegerBits = 24
```

Comment:
Where does the number `24` come from? Is it something that's determined by vamp-ir. Will it need to be user configurable at some point in the future?

Reply:
That's a good question: how many bits should we support? Should this be configurable by the user? Shouldn't we use different datatypes, because Int is supposed to be unbounded?

Efficiency of VampIR degenerates quickly with increasing number of bits, so I chose a middle ground for now, where you can count up to about eight million but VampIR compilation/proof/verification don't take ages (with 32 bits they would take roughly twice as long).